.github/workflows/release.yml

```diff
@@ -25,6 +25,9 @@ jobs:
         with:
           go-version: '1.23'
 
+      - name: Set up QEMU
```

Comment:
Why do you need qemu?

Reply:
We are building cross platform for docker, we need QEMU

Reply:
Can you please add it as comment